web/annotation_layer_builder.css

```diff
@@ -13,363 +13,351 @@
  * limitations under the License.
  */
 
-:root {
+.annotationLayer {
   --annotation-unfocused-field-background: url("data:image/svg+xml;charset=UTF-8,<svg width='1px' height='1px' xmlns='http://www.w3.org/2000/svg'><rect width='100%' height='100%' style='fill:rgba(0, 54, 255, 0.13);'/></svg>");
   --input-focus-border-color: Highlight;
   --input-focus-outline: 1px solid Canvas;
   --input-unfocused-border-color: transparent;
   --input-disabled-border-color: transparent;
   --input-hover-border-color: black;
   --link-outline: none;
-}
 
-@media screen and (forced-colors: active) {
-  :root {
+  @media screen and (forced-colors: active) {
     --input-focus-border-color: CanvasText;
     --input-unfocused-border-color: ActiveText;
     --input-disabled-border-color: GrayText;
     --input-hover-border-color: Highlight;
     --link-outline: 1.5px solid LinkText;
     --hcm-highligh-filter: invert(100%);
-  }
-  .annotationLayer .textWidgetAnnotation :is(input, textarea):required,
-  .annotationLayer .choiceWidgetAnnotation select:required,
-  .annotationLayer
-    .buttonWidgetAnnotation:is(.checkBox, .radioButton)
-    input:required {
-    outline: 1.5px solid selectedItem;
-  }
 
-  .annotationLayer .linkAnnotation:hover {
-    backdrop-filter: var(--hcm-highligh-filter);
+    .textWidgetAnnotation :is(input, textarea):required,
+    .choiceWidgetAnnotation select:required,
+    .buttonWidgetAnnotation:is(.checkBox, .radioButton) input:required {
+      outline: 1.5px solid selectedItem;
+    }
+
+    .linkAnnotation:hover {
+      backdrop-filter: var(--hcm-highligh-filter);
+    }
+
+    .linkAnnotation > a:hover {
```

Comment:
this one could be gathered with the previous one.
That said I'd be in favor of doing that in a follow-up.

Reply:
I've done this in a separate commit, thank you!